engines/experiment/plugin/rpc/manager/rpc_client.go

```diff
@@ -0,0 +1,71 @@
+package manager
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/gojek/turing/engines/experiment/manager"
+	"github.com/gojek/turing/engines/experiment/plugin/rpc/shared"
+)
+
+// rpcClient implements ConfigurableExperimentManager interface
+type rpcClient struct {
+	shared.RPCClient
+}
+
+func (c *rpcClient) Configure(cfg json.RawMessage) error {
+	return c.Call("Plugin.Configure", cfg, new(interface{}))
+}
+
+func (c *rpcClient) GetEngineInfo() (resp manager.Engine) {
+	err := c.Call("Plugin.GetEngineInfo", new(interface{}), &resp)
+	if err != nil {
+		// err should be propagated upstream, but it's currently not
```

Comment:
Should we change the function signature of `GetEngineInfo` to return an error instead ?

Reply:
Yup, perhaps it's a better way to deal with it. Addressed